File: Data Structures & Algorithms/concatenation-of-array/submission-7.rs
impl Solution {
    pub fn get_concatenation(nums: Vec<i32>) -> Vec<i32> {
        let n = nums.len();
        let mut ans = vec![0; 2*n];
        for (i, e) in nums.into_iter().enumerate() {
            ans[i] = e;
            ans[i + n] = e;
        }
        ans
    }
}

File: Data Structures & Algorithms/max-consecutive-ones/submission-7.py
class Solution:
    def findMaxConsecutiveOnes(self, nums: List[int]) -> int:
        count = 0
        maxCount = 0
        for i in nums:
            if i == 1:
                count += 1
            else:
                if count > maxCount:
                    maxCount = count
                count = 0
        if count > maxCount:
            maxCount = count
            
        return maxCount